packages/assets-controllers/src/TokensController.ts

```diff
@@ -457,8 +457,15 @@ export class TokensController extends BaseController<
 
     try {
       incomingDetectedTokens.forEach((tokenToAdd) => {
-        const { address, symbol, decimals, image, aggregators, isERC721 } =
-          tokenToAdd;
+        const {
```

Comment:
Can we add a test for this change? Apologies if it's being tested implicitly — it's not clear to me.

Reply:
Updated the existing test of 'addTokens', you were right with was not being test properly!